keras_cv/losses/serialization_test.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import inspect
+
+import tensorflow as tf
+from absl.testing import parameterized
+
+from keras_cv import core
+from keras_cv import losses as cv_losses
+
+
+def exhaustive_compare(obj1, obj2):
```

Comment:
Instead of copy-pastaing this, let's make it a shared test util.

Reply:
Done